IQSS/dataverse
# 10112-ProvIT-enable-fix

**What this PR does / why we need it**:
The ProvIT test is not working correctly due to the feature not being enabled.

**Which issue(s) this PR closes**:

Closes #10112

**Special notes for your reviewer**:
This change will enable the feature and delete the setting at the end of the test.

**Suggestions on how to test this**:

1. Check out the repo
2. Start docker, mvn -Pct clean package docker:run
3. Run the test, mvn test -Dtest=ProvIT

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:
N/A

**Is there a release notes update needed for this change?**:
N/A

**Additional documentation**:
N/A
